Source/Core/Resource.js

```diff
@@ -1357,9 +1357,11 @@ Resource.prototype._makeRequest = function (options) {
 
   return promise
     .then(function (data) {
+      request.cancelFunction = undefined;
```

Comment:
You can add the same comment here as well (one comment for both here and below is fine).

Thanks!